src/linters/ArcanistESLintLinter.php

```diff
@@ -23,7 +23,7 @@ public function getInfoDescription() {
     }
 
     public function getVersion() {
-        $output = exec('eslint --version');
+        list($output) = execx('%C --version', $this->getExecutableCommand());
 
         if (strpos($output, 'command not found') !== false) {
```

Comment:
fwiw i copied this from the phabricator revision that we stole this code from: https://secure.phabricator.com/D12198#inline-49148